### PR TITLE
chore: support Date ASC sorting MP

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves https://artsyproduct.atlassian.net/browse/CX-3575

Add `date-asc` sorting to metaphysics

### CODE REVIEW

File: src/schema/v2/auction_result.ts [hunk 1]
```diff
@@ -37,6 +37,9 @@ export const AuctionResultSorts = {
       DATE_DESC: {
         value: "-sale_date",
       },
+      DATE_ASC: {
+        value: "-sale_date_asc",
```
Comment (dariakoko) [inline src/schema/v2/auction_result.ts:41]: **Q:** Where is the value `-sale_date_asc` defined?

Comment (MounirDhahri) [reply src/schema/v2/auction_result.ts:41]: I will be defining it in Diffusion in a bit 👀 
I am drafting the PR now

Comment (dariakoko) [reply src/schema/v2/auction_result.ts:41]: Oki :)

Comment (joeyAghion) [reply src/schema/v2/auction_result.ts:41]: According to our usual conventions, ascending sale date would correspond with a `sale_date` value. This `-` and `...asc` seem to be in conflict.

Comment (joeyAghion) [reply src/schema/v2/auction_result.ts:41]: Nevermind! I see now this is updated in https://github.com/artsy/metaphysics/pull/4926.

Comment (MounirDhahri) [reply src/schema/v2/auction_result.ts:41]: Thanks Joey 🙏 and @lordkiz

**Pull request merged**